projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/featured-image/index.tsx

```diff
@@ -168,22 +169,37 @@ export default function FeaturedImage( { busy, disabled }: { busy: boolean; disa
 						</div>
 					) : (
 						<div className="ai-assistant-featured-image__content">
-							<img className="ai-assistant-featured-image__image" src={ imageURL } alt="" />
+							{ error ? (
+								<div className="ai-assistant-featured-image__error">
+									{ __( 'An error occurred while generating the image. Try Again', 'jetpack' ) }
+									{ error?.message && (
+										<span className="ai-assistant-featured-image__error-message">
+											{ error?.message }
+										</span>
+									) }
+								</div>
+							) : (
+								<img className="ai-assistant-featured-image__image" src={ imageURL } alt="" />
+							) }
 							<div className="ai-assistant-featured-image__actions">
-								<Button
-									onClick={ handleAccept }
-									variant="secondary"
-									isBusy={ isSavingToMediaLibrary }
-									disabled={ isSavingToMediaLibrary }
-								>
-									{ __( 'Save and use image', 'jetpack' ) }
-								</Button>
+								{ ! error && (
+									<Button
+										onClick={ handleAccept }
+										variant="secondary"
+										isBusy={ isSavingToMediaLibrary }
+										disabled={ isSavingToMediaLibrary }
+									>
+										{ __( 'Save and use image', 'jetpack' ) }
+									</Button>
+								) }
 								<Button
 									onClick={ handleRegenerate }
 									variant="secondary"
 									disabled={ isSavingToMediaLibrary }
 								>
-									{ __( 'Generate another image', 'jetpack' ) }
+									{ error
+										? __( 'Try again', 'jetpack' )
```

Comment:
I am unsure about using the same handler for the "Try again" button, because of the event being tracked there, `jetpack_ai_featured_image_generation_generate_another_image`. Should we track a different event in this case, in the line of `jetpack_ai_featured_image_generation_try_again`?

Reply:
Done in https://github.com/Automattic/jetpack/pull/36709/commits/645ab3b09934ebd6d7157a3006e90350ecba2d0f